react/features/dynamic-branding/middleware.native.js

```diff
@@ -0,0 +1,49 @@
+import { SET_CONFIG } from '../base/config';
+import { CONNECTION_DISCONNECTED, CONNECTION_FAILED } from '../base/connection';
+import { MiddlewareRegistry } from '../base/redux';
+
+import { SET_DYNAMIC_BRANDING_DATA } from './actionTypes';
+import { fetchCustomBrandingData, unsetDynamicBranding } from './actions.native';
+
+
+MiddlewareRegistry.register(store => next => action => {
+    switch (action.type) {
+    case CONNECTION_FAILED: {
```

Comment:
We shouldn't need this. Every time we end a meeting the config is reloaded, so we can unset it if the newly loaded config doesn't have a branding URL for example.